spinalcordtoolbox/reports/assets/_assets/js/main.js

```diff
@@ -34,24 +34,45 @@ $(document).ready(function(){
 
   $('html').keydown( function(evt) {
     var obj = $('#table tr.active');
-    // Arrow down: next subject
-    if (evt.which == 40) {
-      if (obj.length == 0 || obj.text() === "DateDatasetSubjectPathFileContrastFunctionFunction+Args") {
+    // Arrow down: next subject (or j)
+    if (evt.which == 40 || evt.which == 74) {
+      if (obj.length == 0 || obj.text() === "QCDateDatasetSubjectPathFileContrastFunctionFunction+Args") {
         $('#table tr:first-child').click();
       }
       else {
         obj.next().click();
       }
     }
-    // Arrow up: previous subject
-    if (evt.which == 38) {
+    // Arrow up: previous subject (or k)
+    if (evt.which == 38 || evt.which == 75) {
       if (obj.length == 0) {
         $('#table tr:last-child').click();
       }
       else {
         obj.prev().click();
       }
     }
+    // f key (mark "failing" subjects using check, X, !)
+    if (evt.which == 70) {
+      let index = obj[obj.length - 1].getAttribute("data-index");
+      const heavy_check_mark = '\u2705'
+      const heavy_ballot_x = '\u274C'
+      const heavy_excl_mark = '\u26A0\uFE0F'
+      sct_data[index].qc = (
+          sct_data[index].qc === heavy_check_mark
+          ? heavy_ballot_x
+          : sct_data[index].qc === heavy_ballot_x
+            ? heavy_excl_mark
+            : heavy_check_mark
+      );
+      set_download_yml_btn_state(heavy_excl_mark);
+      set_download_yml_btn_state(heavy_ballot_x);
```

Comment:
These calls fail whenever I use `f`.

![Screenshot from 2021-04-20 09-32-09](https://user-images.githubusercontent.com/16181459/115404425-514eb100-a1bb-11eb-9a66-b44776459f9e.png)

This error message occurs when `set_download_yml_btn_state` can't find an element with the requested ID. So, for me, on Firefox, the buttons are indeed not being created.

(This would also explain why the table isn't being refreshed for me.)

Reply:
The reason why the buttons don't exist is because of https://github.com/neuropoly/spinalcordtoolbox/pull/3253#discussion_r616690188.